schema/config-linux.json

```diff
@@ -253,6 +253,12 @@
             "personality": {
                 "type": "object",
                 "$ref": "defs-linux.json#/definitions/Personality"
+            },
+            "timeOffsets": {
+                "type": "object",
+                "additionalProperties": {
+                    "$ref": "defs-linux.json#/definitions/TimeOffsets"
+                }
```

Comment:
this doesn't seem correct.

I've opened a PR: https://github.com/opencontainers/runtime-spec/pull/1193